src/jou_compiler.h

```diff
@@ -75,7 +75,7 @@ struct Token {
     } type;
     Location location;
     union {
-        int int_value;  // TOKEN_INT
+        long long int_value;  // TOKEN_INT
```

Comment:
Presumably this could be `unsigned long long`, as it is meant to hold 64 bits unsigned.

Reply:
maybe later